core/templates/dev/head/css/oppia.css

```diff
@@ -3901,6 +4214,46 @@ md-card.preview-conversation-skin-supplemental-card {
   right: 0;
 }
 
+/*
+  Styles for Setting tab
+*/
+.oppia-settings-container .has-error .form-text {
+  color: #a94442;
+}
+
+.oppia-settings-container .form-text {
+  color: #737373;
+  display: inline-block;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
+.has-error .form-control {
+  border-color: #a94442;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075);
+  box-shadow: inset 0 1px 1px rgba(0,0,0,.075)
+}
+.has-error .form-control:focus {
+  border-color: #843534;
+  -webkit-box-shadow: inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;
+  box-shadow: inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483
+}
+.oppia-settings-container .col-sm-10 {
+  max-width: 100%;
+  padding-right: 0;
+}
+.oppia-settings-container .col-sm-10  > select {
+  width: 85%;
+}
+
+.oppia-advance-feature-card .oppia-on-off-switch-label {
```

Comment:
advanced-features, and why are these separated from the CSS rules above? Put all CSS rules for one thing in one place. Also this seems like it should belong only in the settings tab page, not in the general oppia.css which gets imported everywhere.

Reply:
Done, yes it belongs only to settings page i've moved it there